axis_backend/apps/kpis/repositories/kpi_type_repository.py

```
@@ -0,0 +1,100 @@
+"""Repository for KPIType model data access."""
+from typing import Optional, List, Dict, Any
```

Ruff (0.14.5)

`typing.List` is deprecated, use `list` instead

(UP035)